src/OnboardingSPA/steps/TheFork/index.js

```diff
@@ -78,25 +92,32 @@ const TheFork = () => {
 		);
 
 		window.location.replace( pluginDashboardPage );
-	};
+	}, [] );
+
 	const content = getContents();
 	const navigate = useNavigate();
 
-	const handleMigration = () => {
+	const handleMigration = useCallback( () => {
```

Comment:
So should we be caching this? cause this might be a one time operation and the cache is pretty limited.

Is there any specific reason you did this?

Reply:
I will remove the usecallback, i had added to test something. will remove